### DISCUSSION TITLE
Is there a way to change the size of staves?

### DISCUSSION THREAD

**Discussion by fablau** [opensheetmusicdisplay/opensheetmusicdisplay#1354] [Q&A] [answered]
Hello,
I am trying to find a way to change the size of staves independently from each other. For example, if I have a score for violin and piano, do I have a way to make the violin staff smaller while keeping the piano staff larger?

I couldn't find anything about that in the documentation, and I am wondering if there is a way to accomplish that.

Thanks in advance for any help!

**Comment by sschmidTU** [comment, marked as answer]
This is unfortunately not possible currently, and would also be quite difficult to implement. We'd have to tell Vexflow to use different sizes and scales at different times, calculate it into the layout, scale the margins, and so on and so forth.

**Reply by fablau** [reply]
I understand, and I hope we'll find a way to do it. I am actively working on your source code, so if I come up with a solution, I'll let you know. Thanks again for your reply!

**Reply by sschmidTU** [reply]
Oh, nice, good luck, and of course we would welcome contributions, especially in the form of pull requests!
You'll probably want to take a look at the use of variables like `DEFAULT_NOTATION_FONT_SCALE` in OSMD and Vexflow. Along with setting the height of the staff of course. But as I said, it's not going to be easy!

**Reply by sschmidTU** [reply]
Oh and maybe look into a way to use the `zoom` feature OSMD already has, but for a single staff. Though maybe the staff will end up being shorter than the others then.

**Reply by fablau** [reply]
Good points. Thank you Simon!
